### ISSUE TITLE
Propergator will not put down.

### ISSUE THREAD

**Issue by Cactusdave** [VoidArkMods/QoLPlus#173]
Since the last update of ark, you can not place the propagator down. It will go through the steps but does not place.

Also any update on the propagator locking up and the buttons not working?

Thanks

**Comment by VoidArkMods** [comment]
Wow, that's a nice one!
On SP it seems to work as intended, but on dedicated servers, placing a QoL+ structure places another random structure instead. I even managed to place a structure from a different mod!
Hopefully it's a bug from the big ASA update that is actively being worked on, but I will have a look as well.

**Comment by VoidArkMods** [comment]
After further investigations, it appears to only do that on my test dedicated server when Ark Descended is also installed. All other big mods I tested it with (including Primal Chaos) do not trigger the issue, but the second I add Ark Descended the issue occurs. Not only with the propagator, but also with storage boxes, dedicated storages, etc. Even after reinstalling Ark Descended, both server-side and client-side, the issue can be reproduced.
Are you also using Ark Descended? If not, I suggest you try disabling mods to reduce the scope of the issue.

**Comment by Cactusdave** [comment]
yes the server is using Ark Descended which is having it's own issues right now as well.
